cmake/std/atdm/common/toss3/environment_new.sh

```diff
@@ -0,0 +1,218 @@
+################################################################################
+#
+# Set up env on toss3 (chama and serrano) for ATMD builds of Trilinos
+#
+# This source script gets the settings from the ATDM_CONFIG_BUILD_NAME var.
+#
+################################################################################
+
+if [ "$ATDM_CONFIG_COMPILER" == "DEFAULT" ] ; then
+  export ATDM_CONFIG_COMPILER=INTEL
+fi
+
+echo "Using toss3 compiler stack $ATDM_CONFIG_COMPILER to build $ATDM_CONFIG_BUILD_TYPE code with Kokkos node type $ATDM_CONFIG_NODE_TYPE"
+
+export ATDM_CONFIG_ENABLE_SPARC_SETTINGS=ON
+export ATDM_CONFIG_USE_NINJA=ON
+export ATDM_CONFIG_BUILD_COUNT=8
```

Comment:
@e10harvey, now that we are building on the compute node, we can likely turn this up a good bit to speed up the build.  Looks like those nodes on cts-1 have 36 total cores (18 cores/socket x 2 sockets).  Let's try to increase this to 20 processes and see how that goes.

Reply:
Scratch that.  We need to keep building on the login node due to a 4-hour time limit for the "short" queue on the CTS-1 machines ('eclipse').